testsuite/test_integration.py

```diff
@@ -58,5 +58,4 @@ def feed_page(url):
 
     # .partial.emlx
     partialemlx = docs[ID['partialemlx']]
-    headers = partialemlx['content']['_emailheaders']
-    assert headers['Subject'] == ["Re: promulgare lege"]
+    assert partialemlx['content']['subject'] == "Re: promulgare lege"
```

Comment:
could you also add a few of the above tested mails (maybe the most complex one) here in the integration test?